bin/server

```diff
@@ -80,10 +80,16 @@ const DatastorePlugin = require(`screwdriver-datastore-${datastoreConfig.plugin}
 const datastorePluginConfig = { ...datastoreConfig[datastoreConfig.plugin] };
 
 // Readonly Datastore
-const datastoreROConfig = datastorePluginConfig.readOnly;
+const datastoreROConfig = { ...datastorePluginConfig.readOnly };
 let datastoreRO;
 
 if (datastoreROConfig && Object.keys(datastoreROConfig).length > 0) {
+    if (datastorePluginConfig.dialectOptions) {
```

Comment:
is it better not to set datastorePluginConfig.directOptions if datastoreROConfig already have a dialecctOptions?

Reply:
Thanks! I fixed.